packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -144,6 +146,8 @@ export function makeRumPublicApi(startRumImpl: StartRum, recorderApi: RecorderAp
   const rumPublicApi = makePublicApi({
     init: monitor((initConfiguration: RumInitConfiguration) => strategy.init(initConfiguration)),
 
+    setTrackingConsent: monitor((trackingConsent: TrackingConsent) => strategy.setTrackingConsent(trackingConsent)),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏‏I don't know If you planned to do it later but it could be nice to add a JS doc. Maybe we will have a related guide?

Reply:
I'm fine with starting to write JSDocs on new public APIs! Good idea.